pkg/docker/stats_type.go

```diff
@@ -0,0 +1,115 @@
+// The following code is from https://github.com/fsouza/go-dockerclient/
+// Copyright (c) 2013-2021, go-dockerclient authors
+package docker
+
+import "time"
+
+// Stats represents container statistics, returned by /containers/<id>/stats.
+// See https://goo.gl/Dk3Xio for more details.
+type Stats struct {
```

Comment:
Do we need to maintain the struct in envd?

Reply:
It is also for future support for GPU. Another point I cared about  is that the godocker repo's author has no GPG keys verified sign which could cause potienal danger.